sycl/test/basic_tests/vec_convert.cpp

```diff
@@ -78,9 +78,9 @@ int main() {
   test<float, float, 8, rounding_mode::automatic>(
       float8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f},
       float8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f});
-  test<float, half, 8, rounding_mode::automatic>(
-      float8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f},
-      half8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f});
+  test<double, float, 8, rounding_mode::automatic>(
```

Comment:
Since you are adding separate tests for particular data types, I suggest you to just remove this test case from here - it is anyway duplicated in `vec_convert_f_to_f.cpp`